.eslintrc

```diff
@@ -1,14 +1,27 @@
 {
-  "extends": ["eslint:recommended", "google"],
+  "parser": "babel-eslint",
+  "parserOptions": {
+    "ecmaVersion": 6,
+    "sourceType": "module",
+    "ecmaFeatures": {
+      "jsx": true,
+      "modules": true,
+      "experimentalObjectRestSpread": true
+    }
+  },
+  "extends": [
+    "eslint:recommended",
+    "google"
+  ],
   "env": {
     "node": true,
-    "mocha": true
+    "mocha": true,
+    "es6": true
   },
   "rules": {
-    "no-var": "off",
     "max-len": [
       "warn",
-      80
+      150
```

Comment:
What is the reason to increase the max length from 80 to 150?

Reply:
`return api.get(_makeOptions(options, path('schedules'), data), callback);` 

If `max-len is 80` upper code is saying is very long. but it is not that too long in my opinion. so I set it to `150`

let me know if you have any suggestion for what max-len is good for our codebase.

Reply:
Let go with 100 for now if 80 is too small number but let's not increase it too much.